### ISSUE TITLE
Error with PHP 8.2

### ISSUE THREAD

**Issue by siemendev** [PHP-CS-Fixer/PHP-CS-Fixer#6787]
### Description
Hi folks!
When I switch my php to 8.2, I get the following error message:
```
PHP needs to be a minimum version of PHP 7.4.0 and maximum version of PHP 8.1.*.
Current PHP version: 8.2.3.
To ignore this requirement please set `PHP_CS_FIXER_IGNORE_ENV`.
If you use PHP version higher than supported, you may experience code modif
```

Is there no 8.2 support for php-cs-fixer? The composer.json defines ^8.0 so I would assume that this should work on 8.2.

### Runtime version
I run php-cs-fixer natively on my mac.
```
PHP CS Fixer 3.14.4 (1b3d9db) Oliva by Fabien Potencier and Dariusz Ruminski.
PHP runtime: 8.1.15
```
(command fails when I run it on >=8.2)

### Used command
```
php-cs-fixer -v
```
### Configuration file
none

### Code snippet that reproduces the problem
not needed

**Comment by kubawerlos** [comment]
Hi, highly encouraging to use the search feature.
TL;DR - 8,2 is not currently supported as it takes a huge effort to support each new PHP syntax. If you need it, consider supporting the project [eg with code contribution]. ref [#6714 (comment)](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6714#issuecomment-1359166470)
